portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,500,700|Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kristin Aoki</h1>
+      <img src="/images/last_page-9.png" alt="Image of Kristin Aoki" style="width:30%;display:block;margin-left:auto;
```

Comment:
TL;DR version - Resize your image to be smaller, it's 21MB right now.

I see that this file is close to 21MB. When you publish web content keep in mind that every one of these bytes has to go over the wire, sometimes over slow networks. I see you do have it styled to 30% but the web browser still has to download all 21MB before it can resize it. I suggest resizing the image file itself and reuploading that. You can just resize it and keep it as PNG which will reduce size. You can also use the JPEG format which is itself a compression format. I'm not too concerned about the file format so much as the size of the image.

Let me know if you have questions about this.

Reply:
Nice job providing descriptive alt text, it's important for accessibility.